be/src/olap/row_block.cpp

```diff
@@ -38,18 +38,33 @@ namespace doris {
 
 RowBlock::RowBlock(const TabletSchema* schema) :
         _capacity(0),
-        _schema(schema) {
+        _schema(schema),
+        _columns(nullptr) {
+    if (_schema != nullptr) {
```

Comment:
if _schema is nullptr, all following operations is no-sense.

Reply:
OK